get_dependencies.py

```diff
@@ -5,7 +5,8 @@
 import requests
 
 
-def download(url, filename):
+def download(url):
+    filename = url[url.rfind("/") + 1:url.find("?")]
```

Comment:
This is a bit brittle and could break if we start using other URLs.
We should use the `Content-Disposition` header of the response and, if it doesn't exist, use the basename of the URL.

Reply:
what about the case when the url will not contain the file name?

Reply:
if there's no `Content-Disposition` header and the basename is an empty string, throw an exception (`Couldn't get file name for this URL`, or something like this).

Reply:
okay